x-pack/plugins/ml/server/lib/register_settings.ts

```diff
@@ -30,5 +34,41 @@ export function registerKibanaSettings(coreSetup: CoreSetup) {
         }),
       },
     },
+    [ANOMALY_DETECTION_ENABLE_TIME_RANGE]: {
+      name: i18n.translate('xpack.ml.advancedSettings.enableAnomalyDetectionDefaultTimeRangeName', {
+        defaultMessage: 'Enable default time range for anomaly detection jobs.',
+      }),
+      value: false,
+      schema: schema.boolean(),
+      description: i18n.translate(
+        'xpack.ml.advancedSettings.anomalyDetectionDefaultTimeRangeDesc',
+        {
+          defaultMessage: 'Use a default time range to view anomaly detection jobs.',
+        }
+      ),
+      category: ['Machine Learning'],
+    },
+    [ANOMALY_DETECTION_DEFAULT_TIME_RANGE]: {
+      name: i18n.translate('xpack.ml.advancedSettings.anomalyDetectionDefaultTimeRangeName', {
+        defaultMessage: 'Time filter defaults',
+      }),
+      type: 'json',
+      value: `{
+  "from": "now-15m",
+  "to": "now"
+}`,
+      description: i18n.translate(
+        'xpack.ml.advancedSettings.anomalyDetectionDefaultTimeRangeDesc',
+        {
+          defaultMessage: 'The default time range to view anomaly detection jobs.',
```

Comment:
I suggest we use text closer to what's used for the timepicker:timeDefaults. For example:
```suggestion
          defaultMessage: 'The time filter selection to use when viewing anomaly detection job results.',
```

Reply:
Updated here https://github.com/elastic/kibana/pull/76347/commits/41af491d1fa355395c34966f6c17b1275917b153